cytotable/presets.py

```diff
@@ -153,6 +153,58 @@
                 AND per_nuclei.Metadata_Nuclei_Number_Object_Number = per_cytoplasm.Metadata_Cytoplasm_Parent_Nuclei
             """,
     },
+    "cellprofiler_legacy": {
```

Comment:
I think a name change here is now necessary based on the information we have. Maybe this could be as specific as outlining `cell-health-cellprofiler-to-cyotominer-database`? Calling out the provenance of the data as it moved through the pipelines I feel could be important for later reference.

Reply:
This has been added in further changes.